care/facility/migrations/0480_migrate_facilities.py

```
@@ -0,0 +1,167 @@
+    29: "Geo Admin Read Only",  # DistrictReadOnlyAdmin
+    30: "Geo Admin",  # DistrictAdmin
+    35: "Geo Admin",  # StateLabAdmin
+    39: "Geo Admin Read Only",  # StateReadOnlyAdmin
+    40: "Geo Admin",  # StateAdmin
+}
+
+
+def get_role_for_user_type(user_type):
+    if role_name := USER_TYPE_TO_ROLE.get(user_type):
+        if role := RoleModel.objects.filter(name=role_name).first():
+            return role.id
```

Ruff (0.8.2)

Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)